dev/publish-nightly.sh

```diff
@@ -16,18 +16,18 @@
 # ==============================================================================
 
 set -e
-cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/../
+cd "$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"/../
 
 # This script will build and publish a nightly release of Flower under the condition
 # that at least one commit was made in the last 24 hours.
 # It will rename the package name in the pyproject.toml to from "flwr" to "flwr-nightly".
 # The version name in the pyproject.toml will be appended with "-dev" and the current date.
-# The result will be a release on PyPi of the package "flwr-nightly" of version e.g. 
+# The result will be a release on PyPi of the package "flwr-nightly" of version e.g.
 # "0.1.1.dev20200716" as seen at https://pypi.org/project/flwr-nightly/
 
 if [[ $(git log --since="24 hours ago" --pretty=oneline) ]]; then
     sed -i -E "s/^name = \"(.+)\"/name = \"\1-nightly\"/" pyproject.toml
-    sed -i -E "s/^version = \"(.+)\"/version = \"\1-dev$(date '+%Y%m%d')\"/" pyproject.toml
+    sed -i -E "s/^version = \"(.+)\"/version = \"\1.dev$(date '+%Y%m%d')\"/" pyproject.toml
```

Comment:
I wanted to use `poetry version -s` to read the version from the `pyproject.toml`, but I noticed later that the hyphen is replaced by a dot in the [normalization](https://peps.python.org/pep-0440/#post-release-separators) version. When I read the version of the `pyproject.toml`, the version didn't match the one on PyPi. Because of that, I changed the hyphen to a dot.

Reply:
Now it is also aligned with the example in the comment above.